modules/apps/document-library/document-library-web/src/main/java/com/liferay/document/library/web/internal/display/context/helper/FileEntryDisplayContextHelper.java

```diff
@@ -255,6 +255,10 @@ public boolean isUpdatable() throws PortalException {
 		return false;
 	}
 
+	public boolean isViewUsagesActionAvailable() {
+		return _permissionChecker.isOmniadmin();
```

Comment:
We should be checking not Omniadmin, as that is too restrictive. I think this might need to be either Content Reviewer or maybe a group admin. I have reached out to @austinchiang to confirm the desired admin level. I will update this once I hear back.

Reply:
I heard back from Austin, we should use groupAdmin. the permission checker object should have an easy call for isGroupAdmin.